src/test/java/camp/nextstep/edu/nextstep8/racing/RacingCarTest.java

```diff
@@ -45,5 +46,19 @@ public void nameTest() {
         // when & then
         assertThat(racingCar.getName()).isEqualTo(name);
     }
+
+    @DisplayName(("자동차에 이름이 부여되지 않을 경우 Exception 발생"))
+    @Test
+    public void nameThrownTest() {
+        // given
+        String name = "";
+
+        // when & then
+        assertThatThrownBy(() -> {
+            RacingCar racingCar = new RacingCar(name);
+        }).isInstanceOf(IllegalArgumentException.class)
```

Comment:
```suggestion
        assertThatThrownBy(() ->new RacingCar(name)).isInstanceOf(IllegalArgumentException.class)
```
위와 같이 선언하는것이 가능합니다 !